bftengine/include/bftengine/KeyExchangeManager.hpp

```diff
@@ -42,6 +42,22 @@ class KeyExchangeManager {
     return ReplicaConfig::instance().getkeyExchangeOnStart() ? (publicKeys_.numOfExchangedReplicas() == clusterSize_)
                                                              : true;
   }
+  const std::string kInitialKeyExchangeCid = "KEY-EXCHANGE-";
+  const std::string kInitialClientsKeysCid = "INITIAL-CLIENTS-PUB-KEYS-";
```

Comment:
Thinking about periodic client keys exchange do we need th INITIAL- prefix?

Reply:
the periodic will be from an external client, and it won't get this CID, I want a way to differentiate an initial request because the initial request verifies against the bootstrap content.

Reply:
I'll remove the initial, see the comment on the comment below